simple_salesforce/bulk.py

```diff
@@ -163,8 +165,36 @@ def _get_batch_results(self, job_id, batch_id, operation):
         return result.json()
 
     #pylint: disable=R0913
+    def worker(self, data, job, operation, wait=5):
+        """ Add batches to concurrent worker threads"""
+        result_list = []
+        while True:
+            data = data[0]
```

Comment:
A comment for why this unpacking is needed would be nice

Reply:
Added additional comments on why unpacking was necessary and included a shutdown of the threads once the worker function has processed the batches.